addons/mail/static/src/owl/components/composer/composer.scss

```diff
@@ -149,42 +166,56 @@
                 }
             }
         }
+    } }
 
-    }
+.o_Composer_button.o-open {
+    background-color: gray('200');
 }
 
-.o_Composer_button {
-
-    &:focus {
-        box-shadow: none;
-        background-color: gray('200');
-    }
+.o_Composer_buttonDiscard {
+    border: 1px solid lighten(gray('400'), 5%);
 
-    &.o-open {
-        background-color: gray('200');
+    &.o_Composer_buttonInline {
```

Comment:
This class looks like a modifier, so rename as `.o-<modifier>` (or `.o_<ComponentName>_<elementName>-<modifier>` to prevent mixing rules with a global modifier)